### PR TITLE
week2-homework

### PR DESCRIPTION
**Pull request by Farzanna**

### CODE REVIEW

File: week2/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+const help = require('./help.txt');
+
+const argv = yargs.argv;
+let command = argv._[0];
+
+console.log('Running Command: ', command);
+if (command === 'help') {
+  console.log(help);
+}
+if (command === 'reset') {
```
Comment (gajduk) [inline week2/homework/src/index.js]: I think you are missing an else

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,42 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+console.log('Running app.js');
+
+const fs = require('fs');
+const _ = require('lodash');
+const yargs = require('yargs');
+const todos = require('./todos.js');
```
Comment (gajduk) [inline week2/homework/src/index.js]: code is organized into multiple files 👍

File: week2/homework/src/todos.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    todos.push(todo);
+    saveTodos(todos);
+    return todo;
+  }
+};
+
+// delete a todo item //
+let deleteTodo = title => {
+  let todos = fetchTodos();
+  let filteredtodos = todos.filter(todo => todo.title !== title);
```
Comment (gajduk) [inline week2/homework/src/todos.js]: interesting approach 👍

File: week2/homework/src/todos.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  saveTodos(filteredtodos);
+
+  return todos.length !== filteredtodos.length;
+};
+
+//reset
+let reset = () => {
+  // return fetchTodos(null);
+  fs.writeFileSync('todos-data.json', JSON.stringify([]));
```
Comment (gajduk) [inline week2/homework/src/todos.js]: why not `saveTodos([])`

**Comment by Tjebbee** [comment]
@Farzanna , please take a look at the feedback from @gajduk ! Thanks for reviewing Andrej

**Comment by Farzanna** [comment]
> Great job @Farzanna . You understand how to work with files and how to organize your code well 👍

Hi @gajduk thank you for your time and your feedback, I'm still struggling with some parts of node.js hope to fix it soon

**Pull request closed without merging**